### DISCUSSION TITLE
Ability to hide top bar on Cardiac Monitor, Defibrillator, 12 Lead ECG, ETC.

### DISCUSSION THREAD

**Discussion by Trinculo54** [tanjera/infirmary-integrated#202] [Ideas]
The title explains what the idea is, but it would be nice to be able to hide the top bar on the simulators for a little more realism.

**Comment by tanjera** [comment]
Great idea! I added it as an issue because that's my running To-Do list. I plan to hammer out some programming over winter break so my ETA on this and other improvements may be about 2 months from now. It may be implemented in [v 2.6](https://github.com/tanjera/infirmary-integrated/releases/tag/v2.6) though that version is nearly done cooking, but I think it can fit this in it!

Thank you for the feedback!
